pkg/errors/errors.go

```diff
@@ -178,6 +178,9 @@ var (
 
 	//ErrNotManifestContentDetected is raised when cannot load any field accepted by okteto manifest doc
 	ErrNotManifestContentDetected = errors.New("couldn't detect okteto manifest content")
+
+	//ErrGithubMissingBusinessEmail is raised when the user does not have a business email
```

Comment:
NIT: I'd add this error on `pkg/okteto/errors` since is an error that is only thrown by that package